azure-pipelines.yml

```diff
@@ -21,7 +21,7 @@ resources:
     - repository: commonTemplates
       type: git
       name: pipelines-yaml-templates
-      ref:  refs/tags/v1.0.11
+      #FIXME: ref:  refs/tags/v1.0.11
```

Comment:
This PR is now draft. 
Please take a look at https://dev.azure.com/sonarsource/DotNetTeam%20Project/_git/pipelines-yaml-templates?path=%2Fset-azp-variables-steps.yml
If it's OK, I'll add tag there and reference it here before merging.

Reply:
it looks fine. Once you add the tag, please also send an email to the SonarLint team as they are using AZP for all SL flavors

Reply:
FIXME is fixed, I'll ping SL once thisone is merged.